net/tcp/tcp_timer.c

```diff
@@ -182,7 +182,7 @@ void tcp_timer(FAR struct net_driver_s *dev, FAR struct tcp_conn_s *conn,
 
               /* Notify upper layers about the timeout */
 
-              result = tcp_callback(dev, conn, TCP_TIMEDOUT);
+              tcp_callback(dev, conn, TCP_TIMEDOUT);
```

Comment:
is this correct? what did clang-check say?

Reply:
ah, I see the commit message. the fact that result is never read probably indicates that it should be read. not assigning the result is not the correct solution if so.

Reply:
see the commit message for the clang output.
i read the code and i agreed with clang about this.

Reply:
ok, tcp_timer does not return errors so I guess it is OK